docs/dev-test-env.md

```diff
@@ -37,7 +46,15 @@ The operator can be built and packaged into an image with:
     IMAGE="quay.io/thoth-station/cnbi-operator"
     make docker-build docker-push IMG="$IMAGE:$IMAGE_VERSION"
 
-The deployment of the operator into the cluster is TBD - it would involve a new CNBi component in https://github.com/operate-first/apps/tree/master/kfdefs/overlays/osc/osc-cl1
+Once the new version image is updated it should be updated in the [manifests](https://github.com/thoth-station/meteor-operator/blob/main/config/manager/kustomization.yaml#L16) for smooth deployment in cluster.
```

Comment:
I believe we should reword that somehow because direct modification of this file is not expected. The version is controlled by the [Makefile](https://github.com/thoth-station/meteor-operator/blob/7206f883dae5dd99385f7350016652bb028e7e99/Makefile#L6)